<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="headLabel">
      <property name="text">
       <string>Шапка</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="authPage">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>90</y>
          <width>221</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Аутентификация</string>
        </property>
       </widget>
       <widget class="QPushButton" name="enterButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>420</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Вход</string>
        </property>
       </widget>
       <widget class="QPushButton" name="exitButton">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>420</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>170</y>
          <width>631</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="loginlabel">
             <property name="text">
              <string>Логин:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="loginEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="passLabel">
             <property name="text">
              <string>Пароль:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="fastEnterButton">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>420</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Автозаполнение</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="adminPage">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>0</y>
          <width>271</width>
          <height>531</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>88</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="analysisButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Рассчет и анализ текущей ситуации</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="orderButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Выдача указаний</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="returnButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Возврат</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>88</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="analysisPage">
       <widget class="QTextBrowser" name="logBrowser">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>421</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="connectionButton">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>40</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Подключение к серверу</string>
        </property>
       </widget>
       <widget class="QPushButton" name="returnButton_2">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>310</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Возврат</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="typeLine">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>452</y>
          <width>711</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="sendButton">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>450</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Отправить</string>
        </property>
       </widget>
       <widget class="QListWidget" name="usersListWidget">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>50</y>
          <width>261</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="logLoadButton">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>110</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Выгрузить лог</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="saveHistoryCheckBox">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>370</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Сохранять историю</string>
        </property>
       </widget>
       <widget class="QPushButton" name="analysisButton_2">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>170</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Анализ ситуации</string>
        </property>
       </widget>
       <widget class="QPushButton" name="analysisButton_3">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>240</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>График статистики</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>410</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="reportCheckButton">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>500</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Проверить</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="plotPage">
       <widget class="QCustomPlot" name="customPlot" native="true">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>781</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="updateButton">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>160</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>220</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Возврат</string>
        </property>
       </widget>
       <widget class="QComboBox" name="plotTypeSelector">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>110</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
